app.R

```diff
@@ -10,35 +10,89 @@ library(rmarkdown)
 library(rprojroot)
 source("src/r/model.R")
 
-ui <- shinyUI(fluidPage(
-  fluidRow(column(1),
-           column(10, h1("Prevalence of Common Mental Health Disorders among people aged 16 to 74,\n in England, by NHS Region, 2014/15")),
-            column(1)),
-  fluidRow(column(1),
-           column(10, sidebarPanel( 
+format_tab <- function(title, header, region_no, map_no, chart_no, narrative_no, metadata_url_no) {
+  tabPanel(title, 
+           fluidRow(column(1), column( 10,h1(header)),
+                    column(1)),
+           fluidRow(column(1), column( 10,sidebarPanel( 
              tags$style(type='text/css', ".selectize-input { font-size: 20px;} .selectize-dropdown { font-size: 20px;}"),
-          
-    selectInput('region', label = h3('Please select an NHS region'), model_outputs[[2]]$Parent.Name))),
-    column(1)),
-  fluidRow(column(5, plotOutput("map", height = "600")),
-           column(7, plotOutput("chart", height = "500", width = "900"))),
-  fluidRow(column(1),
-           column(10,(h2(textOutput("narrative")))),
-          column(1)),
-  fluidRow(column(1),
-           column(10, h3("For more information on this dataset click",
-                         a("here", href= "https://github.com/ONSdigital/mental-health-dashboard/blob/master/src/r/data/Metadata.md", target="_blank"), ".")),
-           column(1)))
+             selectInput(region_no, label = h3('Please select an NHS region'), model_outputs1[[2]]$Parent.Name)))),
+           fluidRow(column(5, plotOutput(map_no,height = "700")),
+                    (column(7, plotOutput(chart_no, height = "500", width = "1000")))),
+           fluidRow(column(1), column(10, h2(textOutput(narrative_no)))), column(1),
+           fluidRow(column(1), column(10, h3("For more information on this dataset click",
+                                             a("here", href= metadata_url_no, target="_blank"), "."),column(1))
+           ))
+}
+
+ui <- shinyUI(
+  fluidPage(
+    titlePanel("Mental Health Dashboard"),
+    
+    mainPanel(
+      column(12,
+             tabsetPanel(
+               
+               format_tab("Mental Health Prevalence", 
+                          "Prevalence of Common Mental Health Disorders among people aged 16 to 74,\n in England, by NHS Region, 2014/15",
+                          "region1",
+                          "map1",
+                          "chart1",
+                          "narrative1",
+                          "https://github.com/ONSdigital/mental-health-dashboard/blob/master/src/r/data/Metadata1.md"),
```

Comment:
I assume the metadata links don't work just because they don't exist until we merge? Need to remember to check these.